source/uri-options/uri-options.rst

```diff
@@ -36,6 +36,28 @@ document are to be interpreted as described in
 **Specification**
 -----------------
 
+Conflicting TLS options
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Per the `Connection String spec <https://github.com/mongodb/specifications/blob/master/source/connection-string/connection-string-spec.rst#repeated-keys>`_,
+the behavior of duplicates of most URI options is undefined. However, due
+to the security implications of certain options, drivers MUST raise an
+error to the user during parsing if any of the following circumstances
+occur:
+
+1. Both ``tlsInsecure`` and ``tlsAllowInvalidCertificates`` appear in the 
+   URI options.
+2. Both ``tlsInsecure`` and ``tlsAllowInvalidHostnames`` appear in the 
+   URI options.
+3. All instances of ``tls`` and ``ssl`` in the URI options do not have the
+   same value. For ease of implementation, drivers MAY fulfill this
```

Comment:
I think it would be weird an annoying if "tls=true&ssl=true" raised an exception.

Reply:
Sorry, that comment wasn't super actionable. Let's require your original idea, the first sentence of bullet 3.

Reply:
Sounds good; I'll update it to remove the second sentence